test/TensorFlow/dynamic_compilation_irgen.swift

```diff
@@ -0,0 +1,22 @@
+// RUN: %target-swift-frontend -emit-ir -Xllvm -tf-dynamic-compilation -Xllvm -debug -Xllvm -debug-only -Xllvm irgensil %s 2>&1 | %FileCheck %s
+
+import TensorFlow
+
+class Dynamic {
+  static let float = 1.0
+}
+
+public func unknownAttribute() {
+  let x: TensorHandle<Float> = #tfop("Dummy1", value$tensor: Dynamic.float)
+  _hostOp(x)
+  // CHECK-LABEL: IRGen for graph_op: Dummy1
+  // CHECK-NEXT: operand: ,ivalue$tensor
+  // CHECK-NEXT: end operands
+}
+
+public func knownAttribute() {
+  let x: TensorHandle<Float> = #tfop("Dummy2", value$tensor: 1.0)
+  _hostOp(x)
+  // CHECK-LABEL: IRGen for graph_op: Dummy2
```

Comment:
do we want to CHECK the operand in the graph_op in the test?

Reply:
CHECK the `value$tensor` attribute operand? It doesn't show up in the IRGen debug output now, because I only print out the dynamic ones. I could add debug output for static attrs, but it doesn't seem worth it since this test should go away soon and lots of other tests already test that static attrs make it to lowering correctly.